### PR TITLE
Doc update

### PR DESCRIPTION
**Pull request by panll**
Documentation update for v1.1 release

### CODE REVIEW

**Comment by ligiabernardet** [comment]
Thank you for the updates. They look good to me (pending action on @JulieSchramm 's comment).
Note that there will be additional updates coming for the App User's Guide once the following issues are settled: a) How the App will be initialized and b) Build system updates.

**Comment by panll** [comment]
> No files should be committed to the build directory.

The files under build directory are removed. Thanks! @JulieSchramm

File: docs/UsersGuide/source/faq.rst [hunk 1]
```diff
@@ -244,6 +244,18 @@ and restart the model for 24 hours simulation:
 
 How do I change a namelist option for chgres_cube or the model?
 ===============================================================
+From the case directory running ./preview_namelists will generate the namelists for the run.  This is normally run by case.submit, but you can also run it from the command line after running the command case.setup.   Run it once before editing user_nl_ufsatm and examine input.nml to see the default value, then edit user_nl_ufsatm and run it again to see the change.
+
+Typical usage of preview_namelists is simply:
+./preview_namelists
+
+The input.nml will be generated under the directory CaseDocs,
```
Comment (JulieSchramm) [inline docs/UsersGuide/source/faq.rst:252]: Commands, file and variable names need to be in `` `` for consistent formatting.

File: docs/UsersGuide/source/faq.rst [hunk 2]
```diff
@@ -293,6 +305,23 @@ or the case subdirectory ``CaseDocs/``.
 Some variables are tied to xml in the case and can only be changed via the
 ``xmlchange`` command. Attempting to change them by editing the file
 ``user_nl_ufsatm`` may generate an error.
+The parameters that need to be changed via xmlchange are defined in namelist_definition_ufsatm.xml
```
Comment (JulieSchramm) [inline docs/UsersGuide/source/faq.rst:308]: xmlchange and namelist_definition_ufsatm.xml should be in double backticks `` ``.

File: docs/UsersGuide/source/faq.rst (hunk 1 above)
Comment (JulieSchramm) [reply docs/UsersGuide/source/faq.rst:252]: Double back tics were interpreted out of comment. Commands, file and variable names need to be in double back tics for consistent formatting.

Comment (panll) [reply docs/UsersGuide/source/faq.rst]: Changed. Thanks!

File: docs/UsersGuide/source/faq.rst (hunk 2 above)
Comment (panll) [reply docs/UsersGuide/source/faq.rst]: Changed. Thanks!

File: docs/UsersGuide/source/faq.rst [hunk 3]
```diff
@@ -244,6 +244,21 @@ and restart the model for 24 hours simulation:
 
 How do I change a namelist option for chgres_cube or the model?
 ===============================================================
+From the case directory running ``./preview_namelists`` will generate the namelists for the run.  This is normally run by case.submit, but you can also run it from the command line after running the command case.setup.   Run it once before editing ``user_nl_ufsatm`` and examine ``input.nml`` to see the default value, then edit ``user_nl_ufsatm`` and run it again to see the change.
```
Comment (JulieSchramm) [inline docs/UsersGuide/source/faq.rst:247]: case.submit and case.setup should be in double tack tics.

Comment (panll) [reply docs/UsersGuide/source/faq.rst]: Updated. Thanks!

File: docs/UsersGuide/source/inputs_outputs.rst [hunk 4]
```diff
@@ -322,7 +338,7 @@ and add
 
 .. code-block:: console
 
-    input_type = "gaussian"
+    input_type = "nemsio"
```
Comment (ligiabernardet) [inline docs/UsersGuide/source/inputs_outputs.rst:341]: This change not correct and this change needs to be reverted (I apologize for my previous guidance).  The existing documentation is correct in this matter:

input_type = "grib2" (App will use files in GRIB2 format) (default)
input_type= "gaussian" (App will use files in NEMSIO format)

Comment (panll) [reply docs/UsersGuide/source/inputs_outputs.rst]: Changed. Thanks!

**Comment by climbfuji** [comment]
@ligiabernardet What is the current status, are you ok with the changes? If yes, please approve and I will merge. Thanks!

**Comment by climbfuji** [comment]
By the way, this branch is called `ufs-release-v1.0` - this is inconsistent with what we do with the rest of the components; the release branch is `release/public-v1`, and we create tags from that branch for `v1.0`, `v1.1`, ... - I think we should fix this inconsistency before merging this PR. @uturuncoglu @jedwards4b

**Comment by climbfuji** [comment]
I created branch release/public-v1 from ufs-release-v1.0 as of today (hash e7028c1 from March 10, 2020) and changed the target for this PR to release/public-v1.

**Comment by ligiabernardet** [comment]
Thank you @climbfuji for creating a branch with the agreed upon nomenclature.

**Pull request merged**